include/BSMPT/minimum_tracer/minimum_tracer.h

```diff
@@ -0,0 +1,830 @@
+// Copyright (C) 2024 Lisa Biermann, Margarete Mühlleitner, Rui Santos, João
+// Viana
+//
+// SPDX-FileCopyrightText: 2024 Lisa Biermann, Margarete Mühlleitner, Rui
+// Santos, João Viana
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#pragma once
+#ifndef INCLUDE_BSMPT_MINIMUM_TRACER_H_
+#define INCLUDE_BSMPT_MINIMUM_TRACER_H_
```

Comment:
The pragma once should be enough

Reply:
Okay, I'll remove this (and only keep pragma once) in the other header files too.